### PR TITLE
Module 3 Math Magician Calculator logic

### PR DESCRIPTION
**Pull request by the-catalystmc**
In this project I:
Copy the files calculator.js and operate.js into a logic/ directory in my project.
Import the files in my Calculator component.
Implement the event handlers I need to use the math logic from calculator.js and operate.js.
Use state and props, pass props to child components, lift the state up.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,37 +1,78 @@
 import '../scss/style.scss';
 import React from 'react';
+import calculate from '../logic/calculate';
```
Comment (DeliceLydia) [inline src/components/Calculator.js]: - [x] The Calculator component should import the `calculate.js & operate.js` files from the logic directory but currently, you are importing `calculate` only please fix that

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
# Hi @the-catalystmc👏,

Good job so far! your calculator is working perfectly and the design is looking like the given one kudos on that! However, there are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (the-catalystmc) [reply src/components/Calculator.js]: @DeliceLydia operate is already imported and used by calculate.js, hence I do not have any use for it in Calculator.js. It throws an error when imported as I will not need to call it.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @the-catalystmc  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**